daemon/mgr/image.go

```diff
@@ -299,6 +308,21 @@ func (mgr *ImageManager) ListImages(ctx context.Context, filter filters.Args) ([
 			continue
 		}
 
+		if len(danglingImages) > 0 {
+			danglingFilter := danglingImages[0]
+			if danglingFilter == trueString {
+				if imgInfo.RepoDigests != nil || imgInfo.RepoTags != nil {
```

Comment:
do we need to judge that `imgInfo.RepoDigests` or `imgInfo.RepoTags` is a empty slice?